### PR TITLE
Add Gpu tutorial for whisper-medium

### PR DESCRIPTION
**Pull request by ch1y0q**

### CODE REVIEW

File: ch_6_GPU_Acceleration/6_3_GPU_Whisper-medium.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+# 6.2 Run Whisper (medium) on Intel GPUs
+
+You can use IPEX-LLM to load Transformer-based automatic speech recognition (ASR) models for acceleration on Intel GPUs. With IPEX-LLM, PyTorch models (in FP16/BF16/FP32) for ASR can be loaded and optimized automatically on Intel GPUs with low-bit quantization (supported precisions include INT4/NF4/INT5/FP8/INT8).
```
Comment (plusbang) [inline ch_6_GPU_Acceleration/6_3_GPU_Whisper-medium.md:3]: Add `FP6` in 'INT4/NF4/INT5/FP8/INT8'

File: ch_6_GPU_Acceleration/6_3_GPU_Whisper-medium.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+For Intel GPUs, **once you have the model in low precision, set it to `to('xpu')`.**
+
+**For INT4 Optimizations (with `load_in_4bit=True`):**
+
+```python
+from ipex_llm.transformers import AutoModelForSpeechSeq2Seq
+
+model_in_4bit = AutoModelForSpeechSeq2Seq.from_pretrained(pretrained_model_name_or_path="openai/whisper-medium",
+                                                  load_in_4bit=True)
```
Comment (plusbang) [inline ch_6_GPU_Acceleration/6_3_GPU_Whisper-medium.md:26]: Please adjust the spacing.

File: ch_6_GPU_Acceleration/6_3_GPU_Whisper-medium.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+```
+
+> [!NOTE]
+> If you have already downloaded the Whisper (medium) model, you could specify `pretrained_model_name_or_path` to the model path.
+
+## 6.2.4 Run Model to Transcribe English Audio
+
+Once you have optimized the Whisper model using IPEX-LLM with INT4 optimization and loaded the Whisper processor, you are ready to begin transcribing the audio through model inference.
+
+Let's start with the English audio file `audio_en.wav`. Before we feed it into Whisper processor, we need to extract sequence data from raw speech waveform:
```
Comment (plusbang) [inline ch_6_GPU_Acceleration/6_3_GPU_Whisper-medium.md:62]: Please add some note here about the source of `audio_en.wav` (such as, from dataset voxpopuli?), and "audio_zh.wav" also needs.

**Review by plusbang — COMMENTED** [review]
Please also update audio files related description in https://github.com/intel-analytics/ipex-llm-tutorial/blob/73b1f882e818e46ec849aac7468ae5b86403c97c/ch_5_AppDev_Intermediate/5_2_Speech_Recognition.ipynb

**Comment by ch1y0q** [comment]
> Please also update audio files related description in https://github.com/intel-analytics/ipex-llm-tutorial/blob/73b1f882e818e46ec849aac7468ae5b86403c97c/ch_5_AppDev_Intermediate/5_2_Speech_Recognition.ipynb

need to update the output accordingly; will do it later

**Comment by plusbang** [comment]
LGTM

**Pull request merged**